docs/creating_component_code_review.md

```diff
@@ -47,8 +47,16 @@ bridge = Bridge(...)
 status = bridge.status()
 ```
 
-### 5. Limit platforms included in initial pull request
-Large pull requests mean there is a larger chance of finding problems that need to be addressed, and more code that needs to be reviewed between every requested change. If your new component will have multiple platforms, try and limit your initial pull request to a single platform. Once the initial component is merged, you can submit additional PRs for the remaining platforms. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new feature in sooner. Pull requests containing large code dumps will not be a priority for review.
+### 5. Make your pull request as small as possible
+Keep a new integration to the minimum functionality needed for someone to get value out of the integration. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new integration/features in sooner. **Pull requests containing large code dumps (> 800 line adds excluding tests) will not be a priority for review and may be closed.**
```

Comment:
800 is also too much. I'd rather we not mention a number at all.

Reply:
I'd love to have some specificity so it's less opinionated.  Do you have an suggestion?

Reply:
No, I don't think we should say a number. That will make it possible to reference the docs, if it's too long but just under what we write here. If we say it's too long in review, it's too long period. We already say they should limit to one platform and don't mix different types of changes in the same PR.

Reply:
Fine, removed.  =)